redirects-list.json

```diff
@@ -26,6 +26,8 @@
   "^/doc/tutorials/get-started(/.*)?$                                                     /doc/start",
   "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-model-files/tutorial",
   "^/doc/tutorials(/.*)?                                                                  /doc/start",
+  "^/doc/start/data-versioning(/.*)?$                                                     /doc/start/data-and-model-versioning",
+  "^/doc/start/data-access(/.*)?$                                                         /doc/start/data-and-model-access",
```

Comment:
```suggestion
  "^/doc/start/data-versioning$                                                           /doc/start/data-and-model-versioning",
  "^/doc/start/data-access$                                                               /doc/start/data-and-model-access",
```

Reply:
How does this work for paths like `/doc/start/data-versioning#subsection`? (Just asking to learn) 

I think it doesn't pass `#subsection` to the redirected page in my version, right?